accounts/forms.py

```diff
@@ -88,6 +88,9 @@ def __init__(self, *args, **kwargs):
             if value is not None and value not in choices:
                 widget.choices.insert(1, [value, value])
 
+        if self.instance.get_ticket() is None:
+            self.fields['is_ukpa_member'].disabled = True
```

Comment:
I read somewhere that it prevents the field being used even if somebody attempts to do so by sending a post request directly to the server.